pkg/kubefed2/federate/disable.go

```diff
@@ -47,11 +47,14 @@ var (
 		--host-cluster-context flag otherwise.`
 
 	disable_example = `
-		# Disable propagation of the Service type
-		kubefed2 federate disable Service
-
-		# Disable propagation of the Service type and delete API resources
-		kubefed2 federate disable Service --delete-from-api`
+		# Disable propagation of the kubernets API type 'Deployment', named
```

Comment:
My apologies for not noticing this on my first pass: s/kubernets/kubernetes/ (also appears on line 54)